packages/caip/src/assetId/assetId.test.ts

```diff
@@ -16,12 +16,9 @@ describe('assetId', () => {
     describe('toAssetId(fromAssetId())', () => {
       it.each([
         ['eip155:1/slip44:60'],
-        ['eip155:3/slip44:60'],
         ['eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d'],
         ['bip122:000000000019d6689c085ae165831e93/slip44:0'],
-        ['bip122:000000000933ea01ad0ee984209779ba/slip44:0'],
```

Comment:
👋🏽  Bitcoin testnet, looks like that was a copypaste job from the CAIP-10 spec https://namespaces.chainagnostic.org/bip122/caip10 nice catch!
<img width="761" alt="image" src="https://github.com/user-attachments/assets/684fa374-acaa-4727-91ab-602f94444490">